app/views/customers/registrations/_form_fields.html.slim

```diff
@@ -0,0 +1,10 @@
+.form-inputs
+  = f.input :name, required: false, autofocus: true, placeholder: 'Your Name', label: false
+  = f.input :customer_account_name, required: true, placeholder: 'Your Business Name', label: false
+  = f.input :email, as: :email, required: true, placeholder: 'Email', label: false
+  = f.input :unconfirmed_phone, as: :tel, required: true, placeholder: 'Phone Number', label: false
+  = f.input :password, required: true, placeholder: 'Your Password', label: false
+  = f.input :password_confirmation, required: true, placeholder: 'Confirm Password', label: false
+  = f.input :tos_accepted, as: :boolean, label: tos_label
```

Comment:
This field is required on Registration, please make sure that it marked as required in UI for this form?